orm/internal/bankexample/state.cosmsos_orm.go

```diff
@@ -0,0 +1,78 @@
+package bankexample
+
+import (
+	context "context"
+
+	protoreflect "google.golang.org/protobuf/reflect/protoreflect"
+
+	ormtable "github.com/cosmos/cosmos-sdk/orm/model/ormtable"
+)
+
+type StateStoreAccessor interface {
+	Open(context.Context) StateStore
+}
+
+func NewStateStoreAccessor() (StateStoreAccessor, error) {
+	panic("TODO")
+}
+
+type StateStore interface {
+	BalanceStore
+	SupplyStore
+}
+
+type BalanceStore interface {
```

Comment:
Can't we hide the complexity of `Open(ctx)` behind `BalanceStore` ?

And just call: `balanceStore.GetBalance(ctx, address, denom)` it would make the API compliant with an external client API.